testsuite/features/init_clients/proxy_container_branch_network.feature

```diff
@@ -24,3 +24,14 @@ Feature: Prepare the containerized branch server for PXE booting
   Scenario: Check the branch network
     Then the "dhcp_dns" host should be present on private network
     And name resolution should work on private network
+
+  Scenario: Let the server know about the new IP and FQDN of the containerized proxy
+    When I follow "Details" in the content area
+    And I follow "Hardware" in the content area
+    And I click on "Schedule Hardware Refresh"
+    Then I should see a "You have successfully scheduled a hardware profile refresh" text
+    When I wait until event "Hardware List Refresh scheduled by admin" is completed
+    And I wait until there is no Salt job calling the module "hardware.profileupdate" on "proxy"
```

Comment:
I would say that in CI, we must use "proxy_container" yet.

Reply:
gosh, i tried to grep, but still missed it

thank you! fixing

Reply:
Ouch... We only use it on 2 lines
```
reposync/allcli_update_activationkeys.feature:    And I select the parent channel for the "proxy_container" from "selectedBaseChannel"
reposync/allcli_update_activationkeys.feature:    And I select the parent channel for the "proxy_container" from "selectedBaseChannel"
```

Reply:
Totally right.
```
def twopence_init(host)
  puts "Initializing a twopence node for '#{host}'."
  raise(NotImplementedError, "Host #{host} is not defined as a valid host in the Test Framework.") unless ENV_VAR_BY_HOST.key? host

  unless ENV.key? ENV_VAR_BY_HOST[host]
    warn "Host #{host} is not defined as environment variable."
    return
  end

  target = "ssh:#{ENV.fetch(ENV_VAR_BY_HOST[host], nil)}"
  node = Twopence.init(target)
  raise LoadError, "Twopence node #{host} initialization has failed." if node.nil?

  $named_nodes[node.hash] = target.split(':')[1]

  # Lavanda library module extension
  # Look at support/lavanda.rb for more details
  node.extend(LavandaBasic)

  return initialize_server(host, node) if host == 'server'

  # Initialize hostname
  hostname, local, remote, code = node.test_and_store_results_together('hostname', 'root', 500)

  unless hostname.empty?
    raise StandardError, "Cannot connect to get hostname for '#{$named_nodes[node.hash]}'. Response code: #{code}, local: #{local}, remote: #{remote}" if code.nonzero? || remote.nonzero? || local.nonzero?
    raise StandardError, "No hostname for '#{$named_nodes[node.hash]}'. Response code: #{code}" if hostname.empty?

    node.init_hostname(hostname)

    fqdn, local, remote, code = node.test_and_store_results_together('hostname -f', 'root', 500)
    raise StandardError, "Cannot connect to get FQDN for '#{$named_nodes[node.hash]}'. Response code: #{code}, local: #{local}, remote: #{remote}" if code.nonzero? || remote.nonzero? || local.nonzero?
    raise StandardError, "No FQDN for '#{$named_nodes[node.hash]}'. Response code: #{code}" if fqdn.empty?

    node.init_full_hostname(fqdn)

    node = process_os_family_and_version(host, fqdn, hostname, node)
    node = process_private_and_public_ip(host, node)
  end

  $node_by_host[host] = node
  $host_by_node[node] = host
  node
end
```

Forget my message. :)

Reply:
No worries. Thanks for the vigilance.